src/libraries/Microsoft.PowerFx.Core/Public/Config/ComposedReadOnlySymbolTable.cs

```diff
@@ -287,5 +287,20 @@ internal override IExternalEntityScope InternalEntityScope
                 return default;
             }
         }
+
+        internal override bool TryGetVariable(DName name, out NameLookupInfo symbol, out DName displayName)
```

Comment:
>override [](http://example.com/codeflow?start=17&length=8)

not a PR blocker, but I wonder if we need a minor refactoring to catch these. IE, make this abstract; and more more things out of ReadOnlySymbolTable.